symphony/lib/core/class.log.php

```diff
@@ -185,7 +204,8 @@ public function popFromLog()
      * @param string $message
      *  The message to add to the Log
      * @param integer $type
-     *  A PHP error constant for this message, defaults to E_NOTICE
+     *  A PHP error constant for this message, defaults to E_NOTICE.
+     *  If null or 0, will be converted to E_ERROR.
```

Comment:
This is overcomplicating it for me. What problem was solved with this? I think we should encourage people to specify something, or be happy with the `E_NOTICE` default.

Reply:
> What problem was solved with this?

Exceptions. By default, they have a code == 0

Reply:
So we have plenty of unknown error in the logs.

Reply:
Gotcha.

Reply:
@brendo wanna approve the PR? I like green check marks.